packages/app-accounts/src/Backup.tsx

```diff
@@ -141,10 +136,14 @@ class Backup extends React.PureComponent<Props, State> {
 
   private onChangePass = (password: string) => {
     this.setState({
-      isPassValid: password.length > 0 && password.length <= 32,
+      isPassValid: this.validatePass(password),
       password
     });
   }
+
+  private validatePass (password: string): boolean {
+    return password.length > 0 && password.length <= 32;
```

Comment:
This could be something useful to add to keyring as well - while you are moving stuff in there, it gets used in backup, restore, create... each time the same.